e2e/tests/api-driven/src/invite-to-pay/mocks/session.json

```diff
@@ -61,7 +61,7 @@
         "applicant.email": "benjamin+itp-payee@opensystemslab.io",
         "property.region": ["London"],
         "application.type": ["ldc.proposed"],
-        "proposal.siteArea": 21.6,
+        "proposal.site.area": 21.6,
```

Comment:
These data value changes are separately related to https://github.com/theopensystemslab/planx-core/pull/642